### DISCUSSION TITLE
Is there any way to get an equivalent chart point from the mouse cursor position?

### DISCUSSION THREAD

**Discussion by Caestille** [Live-Charts/LiveCharts2#657] [Q&A] [answered]
Livecharts had chart.ConvertToChartPoint method, is there anything equivalent in livecharts 2? I'd like to add 'crosshairs' under the mouse cursor with axis sections. I could do this with WPF controls but would prefer the chart handle it if possible

**Comment by beto-rodriguez** [comment, marked as answer]
Crosshair will be supported out of the box in v2, this is already implemented in the dev branch and will be included in the next version 

https://github.com/beto-rodriguez/LiveCharts2/blob/dev/samples/ViewModelsSamples/Axes/Style/ViewModel.cs#L69

and yes, there's a way to convert coordinates:

https://github.com/beto-rodriguez/LiveCharts2/issues/583

**Reply by Caestille** [reply]
Thanks for the pointer!

**Reply by evenuneven** [reply]
Will there be crosshair support in Polar Charts?
